### DISCUSSION TITLE
Wrong vault sorting

### DISCUSSION THREAD

**Discussion by sightkick** [dani-garcia/vaultwarden#3005] [General]
Since my Mac OS updated the Bitwarden Client from 2022.10 to 2022.11 the vaults are not ordered alphabetic but random. On the Bitwarden support page it shows a hint that self hosted versions should be updated to work correctly. 
When is the docker container updated to 2022.11 ?

**Comment by BlackDex** [comment]
The clients have nothing to do with the web-vault versions.
Also, we cant determine the sort order on the server side, since all is encrypted. If sorting is not correct, it probably is something on the client side.

Also, the latest clients could have some small issues with the 1.26.0 release, they should be ok with the main branch `testing` releases though.

**Reply by sightkick** [reply]
Thanks for your quick reply. I hav just now installed the testing release. Fun fact: Now in web vault online view AND in the Mac Client the vaults are already unsorted in random order :-( Have a look at the screenshot from web vault.
![Xnip2022-12-18_07-55-56](https://user-images.githubusercontent.com/57866026/208285591-1ddaca6c-2261-4124-95bd-7590780dd9db.jpg)

**Reply by BlackDex** [reply]
That was too be expected since all clients share the same codebase.

**Reply by sightkick** [reply]
OK... what should I do now... Downgrading the client from the Apple Store should be a problem.

**Reply by BlackDex** [reply]
I have no clue what you can do. This seems to be a bug in the clients.
Probably best to report an issue at Bitwarden regarding this.

**Reply by sightkick** [reply]
Thanks for your quick reply. Tomorrow I will search for my time machine Backup. This should keep an older version of the app. Maybe this will help. In addition to this I will report a bug to bitwarden. Have a nice evening.

**Reply by sightkick** [reply]
OK. I just wanted to raise a ticket and came to the explicit message from to Bitwarden to update the server:
https://bitwarden.com/de-DE/help/november-deprecation-notice/

This is absolutely the time when my Vaults got unsorted. If I tell them that I use vaultwarden on docker they will tell me, that this isn´t supported and I should contact the developer. What do you think?

**Comment by sightkick** [comment]
This the request to get this fixed. Give it a vote :-)
https://community.bitwarden.com/t/sort-the-vaults-in-a-specific-order/47356

**Reply by BlackDex** [reply]
I also created a bug report a while back https://github.com/bitwarden/clients/issues/4263

Forgot to mention it here i see

**Reply by sightkick** [reply]
Much nicer than mine. You do this oftener than me :-) Hope it will help. Yesterday since you updated to 2022.12 the backup of my docker container was my friend :-) Let´s see if the developers will see the requests.
